test/github/mock_api.py

```
@@ -0,0 +1,28 @@
+# needs fastapi
+
+from fastapi import FastAPI
+from pydantic import BaseModel
```

Ruff

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)